tests/flow/test_cms.py

```diff
@@ -177,6 +177,23 @@ def test_merge_extensive(self):
             self.cmd('cms.incrby', 'B', str(i), itemsB[i])
         self.assertOk(self.cmd('cms.merge', 'C', 2, 'A', 'B'))
 
+    def test_overflow(self):
+        large_val = 1024*1024*1024*2 - 1
+        
+        self.cmd('FLUSHALL')
+        self.cmd('cms.initbydim', 'cms', '1000', '5')
+        self.assertEqual([large_val], self.cmd('cms.incrby', 'cms', 'a', large_val))
+        self.assertEqual([large_val], self.cmd('cms.query', 'cms', 'a'))
+        self.assertEqual([large_val * 2], self.cmd('cms.incrby', 'cms', 'a', large_val))
+        self.assertEqual([large_val * 2], self.cmd('cms.query', 'cms', 'a'))
+        
+        # overflow as result > UNIT32_MAX
+        res = self.cmd('cms.incrby', 'cms', 'a', large_val)
```

Comment:
@ashtul please test with few tuples too.

```suggestion
        res = self.cmd('cms.incrby', 'cms', 'a', large_val, 'b', small_val, 'c', large_val)
```

Reply:
added additional elements